argus/tests/cloud/smoke.py

```diff
@@ -352,6 +352,20 @@ def test_static_network(self):
         for nics in (guest_nics, instance_nics):
             for nic in nics:
                 nic["dns6"] = None
+
+        # If os version < 6.2 then ip v6 configuration is not available
+        # so we need to remove all ip v6 related keys from the dicts
+        major, minor = self.introspection.get_instance_os_version()
+        if not (major >= 6 and minor >= 2):
+            for nic in guest_nics:
+                for key in nic.keys():
```

Comment:
Always wrap the keys() call in a list when mutating the dictionary in a for loop, because this will fail in Python 3 with a RuntimeError.